scripts/scripts.js

```diff
@@ -211,6 +213,8 @@ async function loadLazy(doc) {
   sampleRUM.observe(main.querySelectorAll('div[data-block-name]'));
   sampleRUM.observe(main.querySelectorAll('picture > img'));
 
+  integrateMartech(document.body, 'GTM-KNBZTHP');
```

Comment:
the key should be retrieved from place holders and not be hard-coded. the key may also differ between `.page`, `.live` and `localhost`